updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).
+
+[id="updating-clusters-overview-understanding-container-platform-service"]
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an OpenShift Update Service as a hosted service located behind public APIs.
+
+[id="updating-clusters-overview-installing-and-configuring-service"]
+== Installing and configuring the OpenShift Update Service
+xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: You can install and configure the {product-title} update service by using a hosted service located behind public APIs. However, clusters in a restricted network have no way to access the public APIs for update information. For more information, see the following procedures:
+
+* xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+* xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-delete-service[Deleting an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-uninstall[Uninstalling the OpenShift Update Service Operator]
+
+
+[id="updating-clusters-overview-upgrade-channels-and-releases"]
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of {product-title} always installs that minor version. In addition to the minor version, you can select from the `candidate`, `fast`, `stable` and `eus` channel tiers to enable an update strategy that suits the needs of your environment. For more information, see the following:
```

Comment:
Looks like the same text in the 4.10 docs, but "Upgrade channels control only release selection" reads awkwardly to me.  I'd expect something like "Upgrade channels control update recommendations..." or some such.  Or do we define "release selection" as an update-recommendation thing somewhere else?

Reply:
Ack'ed- updated text to `Upgrade channels control update recommendations...`. IMO it would be sufficient for the overview page.